### PR TITLE
I added an api key as a parameter

### PR DESCRIPTION
**Pull request by adrotter**
Note that I wasn't able to get any coordinates from the addresses provided in the console program, so I am not sure if the program works.  But the URL doesn't throw a key related error when used with a proper key.

### CODE REVIEW

**Comment by adrotter** [comment]
also, the apikey is optional, just leave the string blank if no key is provided.

File: GoogleMaps.LocationServices/GoogleLocationService.cs [hunk 1]
```diff
@@ -5,113 +5,113 @@
...
+            return null;
+        }
+
+        /// <summary>
+        /// Translates a Latitude / Longitude into an address using Google Maps api
+        /// </summary>
+        /// <param name="latitude"></param>
+        /// <param name="longitude"></param>
+        /// <returns></returns>
+        public AddressData GetAddressFromLatLang(double latitude, double longitude, string apikey)
```
Comment (LifeAlgorithm) [inline GoogleMaps.LocationServices/GoogleLocationService.cs]: Can add default parameter as ```string apikey = "" ```

**Comment by sethwebster** [comment]
Sorry for the delay. @adrotter, can you resolve these conflicts? Thanks!

**Comment by adrotter** [comment]
I'm sorry @sethwebster, what conflicts?  Merge conflict?

**Comment by sethwebster** [comment]
Yes; there are conflicts.



Seth Webster
@sethwebster <http://twitter.com/sethwebster> | web site
<http://sethwebster.com/> | photography <http://photos.sethwebster.com/> |
IMDB <http://www.imdb.com/name/nm0917003/?ref_=fn_al_nm_1> | +1.646.350.0739
(c)

**Comment by sethwebster** [comment]
Since the apiKey would most likely be used across any of the requests, why not add it in the constructor or as a settable property, @adrotter ?

**Comment by adrotter** [comment]
I'm going to close and open a new pull request since you've done a lot of work since this one and I want to start from your current code.

**Comment by sethwebster** [comment]
Great thx.

**Pull request closed without merging**